operator/flags.go

```diff
@@ -178,7 +178,7 @@ func init() {
 	flags.Bool(option.EnableIPv4Name, defaults.EnableIPv4, "Enable IPv4 support")
 	option.BindEnv(option.EnableIPv4Name)
 
-	flags.String(operatorOption.ClusterPoolIPv4CIDR, "",
+	flags.StringSlice(operatorOption.ClusterPoolIPv4CIDR, []string{},
```

Comment:
The change of field type doesn't seem to be included in this commit, did you forget to run `git add`?

Reply:
Not necessary. The code actually already is read as a string slice. See https://github.com/cilium/cilium/pull/17780#discussion_r742735839 and https://github.com/cilium/cilium/blob/858fca7e2cfa78d3b5faa8d28086c2d8581fde8c/operator/option/config.go#L376

Reply:
I see, thanks for clarifying.